src/website/src/pages/ec/layout/grid/index.mdx

```diff
@@ -3,29 +3,98 @@ title: Grid
+- The column classes **express the number of columns to be engaged out of the maximum**, which is 12 per row (e.g. an element invoking `.ecl-col-2` occupies two columns)
+- **Container widths and breakpoints are expressed in pixels**
+- Each column has **horizontal padding (gutter) for regulating the space between** them
+- **The gutters are expressed in rems**
+- **The width of the columns is always proportional to the container width**, which is fixed (except for viewports <480px), and they are expressed in percentages
+- The rows wrap around the columns, allowing you to specify the amount of columns content wraps around
 
-Moreover, `.ecl-col, .ecl-col-sm, ...` without the number of columns is not allowed. You have to **explicitly specify** the number of columns (out of 12).
+## ECL Grid Options
 
-## Breakpoints
+The grid's breakpoints are established on the minimum width media queries, therefore the are applied on the targeted breakpoint and all those above it.
 
-There are four breakpoints in our responsive grid
+| viewport        | Extra small (≥ 0px) | Small (≥ 480px) | Medium (≥ 768px) | Large (≥ 996px) | Extra large (≥ 1140px) |
+| --------------- | ------------------- | --------------- | ---------------- | --------------- | ---------------------- |
+| container width | 100%                | 480px           | 768px            | 996px           | 1140px                 |
+| class prefix    |  `.ecl-col-`        | `.ecl-col-sm-`  | `.ecl-col-md-`   | `.ecl-col-lg-`  | `.ecl-col-xl-`         |
 
-<DisplayTokens tokens={tokens} category="breakpoints" />
+## ECL Grid Examples
 
-## Grid
+<Playground
+  system="ec"
+  selectedKind="layout-grid"
+  selectedStory="2-3-4-6-12"
+  showFrame
+>
+  <Example234612 />
+</Playground>
 
-The ECL grid uses a 12 column as the basic underlying structure for all web pages. The width of the columns is adaptable to the width of the screen size.
+<Playground
+  system="ec"
+  selectedKind="layout-grid"
+  selectedStory="6-4-2"
+  showFrame
+>
+  <Example642 />
+</Playground>
 
-Gutters are the gaps between the columns. Our grid contains 11 gutters of 2rems (between each column) and 1 more gutter divided in half between the first and the last column.
+<Playground
+  system="ec"
+  selectedKind="layout-grid"
+  selectedStory="2-8-2"
+  showFrame
+>
+  <Example282 />
+</Playground>
 
-<DisplayTokens tokens={tokens} category="grid" />
+<Playground
+  system="ec"
+  selectedKind="layout-grid"
+  selectedStory="3-3-6"
+  showFrame
+>
+  <Example336 />
+</Playground>
 
-![An image of a grid](https://inno-ecl.s3.amazonaws.com/media/images/EC/Grid/Grid.svg)
+## Grid offset
+
+When using the ECL grid, there is the ability to offset columns. You can offset columns from left to right by invoking the `.ecl-offset-*` classes (where \* = the number of columns). These classes simply increases the left margin by \* number of columns.
```

Comment:
Well, should be "These classes simply increase"..

Reply:
good catch :) I've fixed it in my last commit